### DISCUSSION TITLE
How to disable image links

### DISCUSSION THREAD

**Discussion by hultqvist** [dotnet/docfx#10373] [Q&A] [answered]
Using the ``default`` and ``modern`` templates, every image added becomes clickable, linking directly to the same image source.

    ![Happy](happy.png)

The raw output from the web server looks as expected:

    <p><img src="happy.png" alt=""></p>

But the final DOM after JS modifications looks like this:

    <p><a target="_blank" rel="noopener noreferrer nofollow" href="https://example.com/happy.png"><img src="/happy.png" alt=""></a></p>

Is there a way to disable this?

**Comment by filzrev** [comment]
I thought these behaviors can be suppressed by configuring `showLightbox` option.
Currently these option can't be configured with docfx metadata.

It need to create `custom docfx template`. 
And need to create custom `main.js` file.
https://github.com/dotnet/docfx/issues/9415

**Comment by hultqvist** [comment, marked as answer]
Based on the pointers from @filzrev I created a template at ``templates/my``

Added it to `docfx.json`

```
{
  "build": {
    "template": [
      "default",
      "modern",
      "templates/my"
    ]
  }
}
```

Added the option to `templates/my/main.js`

```
export default {
    showLightbox: (element) => false,
}
```
